src/test/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContextTest.java

```diff
@@ -38,6 +40,8 @@ public class WebRequestProcessingContextTest {
 	@Test
 	public void throwsExceptionIfPathVariableDoesntExist() {
 		MockWebRequest req = new MockWebRequest("/customers/888/orders/99");
+		setPathVariables(req, pathVariables(entry("customerId", "888"), entry("orderId", "99")));
```

Comment:
If path variable should not exists, why are you set path variables in this test?

Reply:
+1

Reply:
this test is about throwing exception when trying to get path variable with name "notExisting", not about path variables not existing at all